src/file.cpp

```diff
@@ -62,7 +73,7 @@ static Core* readRuntimeObject(BinaryReader& reader,
 				// Still couldn't find it, give up.
 				fprintf(
```

Comment:
On a related note, I think it might be beneficial if the rive-cpp lib provided a logging function callback.
A default one could be registered, and it could use the current approach, outputting to stderr.